.travis.yml

```diff
@@ -1,16 +1,28 @@
 language: python
+
 install:
-    - git clone --depth 1 git://github.com/astropy/ci-helpers.git
-    - source ci-helpers/travis/setup_conda.sh
+  - git clone --depth 1 git://github.com/astropy/ci-helpers.git
+  - source ci-helpers/travis/setup_conda.sh
+  - source build_tools/travis/install.sh
+
 script:
-    - pytest --cov=./
-    - codecov
+  - pytest --cov=./
+  - codecov
+
 matrix:
   include:
     - name: "Python 3.6 with minimum dependencies (no pandas and python-Levenshtein)"
       python: "3.6"
-      env: CONDA_DEPENDENCIES='distance sklearn joblib numpy scipy requests pytest-cov codecov coverage'
+      env:
+        - CONDA_DEPENDENCIES='distance joblib numpy scipy requests pytest-cov codecov coverage'
+        - SCIKIT_LEARN_VERSION='stable'
     - name: "Python 3.8 with pandas"
       python: "3.8"
-      env: CONDA_DEPENDENCIES='python-Levenshtein distance sklearn joblib numpy scipy requests pytest-cov codecov coverage pandas'
-
+      env:
+        - CONDA_DEPENDENCIES='python-Levenshtein distance joblib numpy scipy requests pytest-cov codecov coverage pandas'
+        - SCIKIT_LEARN_VERSION='stable'
+    - name: "Python 3.8 with scikit-learn nightly-build"
+      python: "3.8"
```

Comment:
I think that 3.9 got released a few days ago. Do you want to see if we can test on 3.9?